install/gke.go

```diff
@@ -46,7 +46,7 @@ func (k *K8sInstaller) gkeNativeRoutingCIDR(ctx context.Context, contextName str
 	result := exec.Command("gcloud", args...)
 	bytes, err := result.Output()
 	if err != nil {
-		return "", fmt.Errorf("unable to execute gcloud %s to extract native routing CIDR: %w", args, err)
+		return "", fmt.Errorf("unable to execute gcloud %s to extract native routing CIDR: %w", strings.Join(args, " "), err)
```

Comment:
Nit:  should probably have quotes around the command here to make it consistent with the `az` output. We could opt for single quotes to avoid having to escape.